{"name": "test_admin_login[case2_用户名错误的登录]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <case.test_member_controller.test_login.Test_DengLu object at 0x00000200EAC80588>\ncase_data = {'body': {'password': '12345678', 'username': 'adminddd'}, 'case_name': 'case2_用户名错误的登录', 'expect': {'code': 500, 'message': 'Bad credentials'}}\nmember_controller_z = <api.admin.member_controller_api.MemberControllerApi object at 0x00000200EACA1710>\n\n    @decorate_log\n    @pytest.mark.parametrize(\"case_data\", data,ids=[data[i]['case_name'] for i in range(len(data))])  # 参数化测试用例\n    \n    def test_admin_login(self,case_data,member_controller_z):\n        # time.sleep(3)\n        print(\"case_data\",case_data)\n        case_body = case_data['body']\n        \"\"\"请求登录测试用例\"\"\"\n        with allure.step('登录步骤'):\n            actual_data = member_controller_z.login(password=case_body['password']\n                           ,username=case_body['username'])\n            \"\"\"断言\"\"\"\n>           assert_api(actual_data,case_data['expect'])\n\ntest_login.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\common\\logger.py:68: in log\n    raise e\n..\\..\\common\\logger.py:64: in log\n    func(*args, **kwargs)\n..\\..\\common\\assert_api.py:36: in assert_api\n    raise e\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nactual_data = {'code': 200, 'data': {'expiresIn': 86399, 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZ...hzvXUFfoiTbv2zyEGdlXLQIQ8IwbjWoQ-StXsrJPqn0eqkIF876kOLk5WtUU6GVEA0sYA7G-M', 'tokenHead': 'Bearer '}, 'message': '操作成功'}\nexpect_data = {'code': 500, 'message': 'Bad credentials'}\n\n    @decorate_log\n    def assert_api(actual_data,expect_data):\n        # Logger.info(\"*************** {}开始执行用例 ***************\".format(expect_data['info']))\n        Logger.info(\"\\r\\nassert:\\r\\n 实际结果：{}，\\r\\n期望结果{}\".format(actual_data, expect_data))\n        try:\n            a = expect_data\n            b = actual_data\n            for i, j in a.items():\n                if i in b.keys():\n>                   assert j == b[i]\nE                   AssertionError\n\n..\\..\\common\\assert_api.py:27: AssertionError"}, "steps": [{"name": "登录步骤", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\case\\test_member_controller\\test_login.py\", line 37, in test_admin_login\n    assert_api(actual_data,case_data['expect'])\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 68, in log\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 64, in log\n    func(*args, **kwargs)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 36, in assert_api\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 27, in assert_api\n    assert j == b[i]\n"}, "steps": [{"name": "响应日志", "status": "passed", "parameters": [{"name": "status_code", "value": "200"}, {"name": "text", "value": "{'code': 200, 'message': '操作成功', 'data': {'token': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4MTk0ODgxLCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiMTBhOWJlOWMtYmM4ZC00MDg4LThiOWEtMmNlMjgyOWZiOTQ0IiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.fvQRXCxBTUUMzQjDuZ6m_DhleKs25RD9_TewidBArq5XkfjaHjVhZ2iaoYnZKmgf6VFm1WoQr2b0PjXApTWZTeUFQyHppOjZWChzvXUFfoiTbv2zyEGdlXLQIQ8IwbjWoQ-StXsrJPqn0eqkIF876kOLk5WtUU6GVEA0sYA7G-M', 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJhdGkiOiIxMGE5YmU5Yy1iYzhkLTQwODgtOGI5YS0yY2UyODI5ZmI5NDQiLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4NzEzMjgxLCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiMmY0NjVhM2ItN2E1MS00MjY1LWE1YmQtMjdlMmFiZGNjZGY2IiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.L6uabOqcRKow2-6XM19pR6anIADPYNwHLt-7-vYv8cEtn7z4_Eb3-gUZ28-6aFr-kZOG4uUUgiGlB_qj3hmka_HAfHQMPCgKnn7rVN-qc9g21c76FTeptaPp9pg3gRCsgfUP7U0k6yzPaiUNOl9ZSreNxYqHT_nNd8a8meNGASM', 'tokenHead': 'Bearer ', 'expiresIn': 86399}}"}], "start": 1648108481217, "stop": 1648108481217}, {"name": "请求日志", "status": "passed", "parameters": [{"name": "url", "value": "'http://121.37.169.128:8201/mall-member/sso/login?platform=test&time=1648108480833&username=admin&password=123456'"}, {"name": "method", "value": "'post'"}, {"name": "body", "value": "{}"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}], "start": 1648108481217, "stop": 1648108481217}], "start": 1648108480833, "stop": 1648108481218}], "attachments": [{"name": "log", "source": "26dbe6fd-7d9a-4ec0-9641-b859e38a971a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_data", "value": "{'case_name': 'case2_用户名错误的登录', 'body': {'username': 'adminddd', 'password': '12345678'}, 'expect': {'message': 'Bad credentials', 'code': 500}}"}], "start": 1648108480833, "stop": 1648108481219, "uuid": "8716eacb-ee28-42c3-8f7d-1ae8af2973e9", "historyId": "0f326f180f219452afd11079cb76c2e0", "testCaseId": "02c7f242c9f09fbdefa4b8f5b820915e", "fullName": "case.test_member_controller.test_login.Test_DengLu#test_admin_login", "labels": [{"name": "feature", "value": "测试member_controller模块"}, {"name": "story", "value": "登录功能"}, {"name": "parentSuite", "value": "case.test_member_controller"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_DengLu"}, {"name": "host", "value": "LAPTOP-9964GMTD"}, {"name": "thread", "value": "50056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_member_controller.test_login"}]}